examples/nas/cream/Cream_zh_CN.md

```diff
@@ -0,0 +1 @@
+[文档](https://nni.readthedocs.io/en/latest/NAS/Cream.html)

```

Comment:
The `/en` in readthedocs should be replaced.

Reply:
I forgot the links outside the folder `zh_CN`... it has been updated.